### PR TITLE
Prepare for crate release

### PR DESCRIPTION
**Pull request by rpkelly**
*Description of changes:*
Prepare tuftool, tough-kms, and tough-ssm for new crate release.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by etungsten — APPROVED** [review]
:hamburger:

**Comment by etungsten** [comment]
Do we need to update `Cargo.lock` somewhere?
From the integ workflow:
```
error: the lock file /home/runner/work/tough/tough/Cargo.lock needs to be updated but --locked was passed to prevent this
If you want to try to generate the lock file without accessing the network, remove the --locked flag and use --offline instead.
```

File: tuftool/Cargo.toml [hunk 1]
```diff
@@ -36,8 +36,8 @@ structopt = "0.3"
 tempfile = "3.3.0"
 tokio = "~1.18"  # LTS
 tough = { version = "0.12.2", path = "../tough", features = ["http"] }
```
Comment (jpculp) [inline tuftool/Cargo.toml]:
```suggestion
tough = { version = "0.12.3", path = "../tough", features = ["http"] }
```
We should probably cut a new release of `tough` as well.

File: tough-kms/CHANGELOG.md [hunk 2]
```diff
@@ -77,7 +84,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Everything!
 
-[Unreleased]: https://github.com/awslabs/tough/compare/tough-kms-v0.3.6...develop
+[Unreleased]: https://github.com/awslabs/tough/compare/tough-kms-v0.4.0...develop
+[0.3.6]: https://github.com/awslabs/tough/compare/tough-kms-v0.3.6...tough-kms-v0.4.0
```
Comment (bcressey) [inline tough-kms/CHANGELOG.md]:
```suggestion
[0.4.0]: https://github.com/awslabs/tough/compare/tough-kms-v0.3.6...tough-kms-v0.4.0
```

File: tuftool/CHANGELOG.md [hunk 3]
```diff
@@ -165,7 +172,8 @@ Major update: much of the logic in `tuftool` has been factored out and added to
 ### Added
 - Everything!
 
-[Unreleased]: https://github.com/awslabs/tough/compare/tuftool-v0.7.2...develop
+[Unreleased]: https://github.com/awslabs/tough/compare/tuftool-v0.8.0...develop
+[0.7.2]: https://github.com/awslabs/tough/compare/tuftool-v0.7.2...tuftool-v0.8.0
```
Comment (bcressey) [inline tuftool/CHANGELOG.md]:
```suggestion
[0.8.0]: https://github.com/awslabs/tough/compare/tuftool-v0.7.2...tuftool-v0.8.0
```

**Review by jpculp — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**